### DISCUSSION TITLE
An odd "contributor" to one of my repos

### DISCUSSION THREAD

**Discussion by autoSteve** [community/community#70079] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a repository with an odd contributor shown. NGUYEN Linh. The repo is autoSteve/acMqtt.

She opened a GitHub account back in 2018, and has never contributed anything since, other than an Instagram worthy profile pic. And certainly has never committed anything to my repo.

None of the tips and suggestions, like renaming the branch have worked, and this Mac is only ever used by me. All commits are mine, with one contribution by Paddy B.

Ms. Nguyen showed up as a "contributor" when I made a hash of setting up GPG signing about six months back. That's all sorted now, but Ms Nguyen has remained ever since.

Anyone got a clue as to what might be going on?

**Comment by DJLink** [comment]
This might be a github bug, if we list contributors only 2 show up, despite that person showing up on the repo main page. 

If we go to the contributors page again, 2 persons show up.

https://github.com/autoSteve/acMqtt/graphs/contributors

Try seeing if you can see them in removing a contributor? https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/removing-a-collaborator-from-a-personal-repository

if you can pinpoint the exact commit where they show up maybe that could be a clue for github team to look into it?

**Reply by autoSteve** [reply]
Yeah, nah. I tied that.

"0 collaborators have access to this repository. Only you can contribute to this repository."

Identifying the specific commit would be hard given the time past since the user showed up, and I can only tie it to setting up GPG signing. At the time I remember messing with those signing settings and seeing Nguyen disappear, only to come back again. There would have been commits involved to test things.

**Reply by DJLink** [reply]
sorry out of ideas, the best way would be to contact github support via this page I think

https://support.github.com/contact/bug-report

**Reply by autoSteve** [reply, marked as answer]
...And, she's now gone from contributors.

Maybe it was to do with finally getting GPG signing working, or me lookig at that again, then some magic eventually running on the back-end.

Thanks for looking, @DJLink.
